changelog/fixed-nightly-warnings-20250611.md

```diff
@@ -0,0 +1 @@
+Fixed warnings that started showing up with 2025-06-11 nightly
```

Comment:
lol please don't changelog this

Reply:
you can put the label to make it happy :)